e2e-tests/insurance/cypress/e2e/journeys/your-buyer/check-your-answers/check-your-answers-page.spec.js

```diff
@@ -52,7 +53,7 @@ context('Insurance - Your buyer - Check your answers - As an exporter, I want to
     cy.corePageChecks({
       pageTitle: CONTENT_STRINGS.PAGE_TITLE,
       currentHref: `${ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`,
-      backLink: `${ROOT}/${referenceNumber}${TRADED_WITH_BUYER}`,
+      backLink: `${ROOT}/${referenceNumber}${BUYER_FINANCIAL_INFORMATION}`,
```

Comment:
Is the back link now conditional depending on the total contract value threshold?

Reply:
no, that page will always be completed